index.html

```diff
@@ -0,0 +1,153 @@
+					<li><a href="#">About</a></li>
+					<li><a href="#">Contact</a></li>
+				</ul>
+				<div class="pull-right quick-links">
+					<div class="header-name">
+						<span>ryan wilson</span>
+					</div>
+					<div class="header-social">
+						<a href="https://www.linkedin.com/in/ryanwilson801"><img src="images/linkedin.png"></a>
+						<a href="https://github.com/ryanadamwilson"><img src="images/github.png"></a>
+					</div>
+				</div>
+			</nav>
+		</header>
+		<div id="introContent">
+			<div class="container-fluid">
+				<div class="row">
+					<div class="col-xs-12">
+						<h1 id="title">Ryan Wilson | Front End Developer</h1>	
+					</div>
+				</div>
+				<div class="row">
+					<div class="col-xs-12">
+						<p class="intro-text">I am a front end developer passionate about building modern and responsive websites.</p>
+						<div class="intro-port" onClick="portfolioBtn();">Check out some cool things I've built</div>
+					</div>
+				</div>
+			</div>
+		</div>
+		<div id="skillsContent">
+			<div class="container-fluid">
+				<div class="row">
+					<div class="col-xs-12 col-md-6 pull-right skills">
+						<h2 class="contentHeader">Skills</h2>
+						<p>I create websites using HTML, CSS, and JavaScript. I most often prefer to work with WordPress, but am I also capable of using any content management system. I additionally focus on responsive design to ensure my sites work well on any screen size and device.</p>
+						<!--<p>I also offer copywriting, proofreading and editing as well as content management services.</p>
+						<p>My role in the creation of your web presence can be anything from a one-time meeting to an ongoing experience with continuous support.</p>-->
+						<p>My Key Skills:</p>
+						<ul id="skillsList">
+							<li>HTML</li>
+							<li>CSS 3</li>
+							<li>JavaScript</li>
+							<li>jQuery</li>
+							<li>WordPress</li>
+							<li>Bootstrap</li>
+							<li>Responsive Design</li>
+						</ul>
+					</div>
+				</div>
+			</div>
+		</div>
+		<div id="projectsContent">
+			<div class="projects-module">
+				<div class="row">
+					<div class="col-sm-6 col-xs-12">
+						<div class="project">
+							<div class="col-xs-3">
+								<div class="circle-thumb fizzbuzz">Fizz</div>
+							</div>
+							<div class="col-xs-9">
+								<div class="project-content">
+									<div class="project-title">FizzBuzz JavaScript</div>
+									<p class="project-para">This project exemplifies basic JavaScript functions to create a game called FizzBuzz. In addition, all of the HTML was generated dynamically using functions, apart from the body and one div tag.</p>
+									<a href="/rwilson/FizzBuzz/"><div class="see-example">View Project</div></a>
+								</div>
+							</div>
+						</div>
+					</div>
+					<div class="col-sm-6 col-xs-12">
+						<div class="project">
+							<div class="col-xs-3">
+								<div class="circle-thumb madlib"><span>MAD </span><span>LIB</span></div>
```

Comment:
Check out the picture I added of a screen shot from my phone. It looks pretty good for mobile design, just one or two thing you might notice, the project circles. 

![](https://lh3.googleusercontent.com/-nHBESvYWHgM/VYt3xlxv7bI/AAAAAAAAEY4/-rg91VbzfsU/w893-h1585-no/2015-06-24.jpg)